### ISSUE TITLE
Bazarr unable to import existing subtitles for multi episodes (e.g. S01E01-E02)

### ISSUE THREAD

**Issue by fabiorzfreitas** [morpheus65535/bazarr#1562]
**Describe the bug**
After selecting subtitles for multi episodes (such as S01E01-E02), Bazarr halts and the whole import queue is not processed.

Additionally, log ends with `TypeError: int() argument must be a string, a bytes-like object or a number, not \'list\`

**Expected behavior**
Bazarr post-processing the uploaded file

**Log**
<details>
    <summary>Traceback from Log section</summary>

```
Traceback (most recent call last):

  File "c:\\program files\\bazarr\\bazarr\\../libs\\flask\\app.py", line 1949, in full_dispatch_request

    rv = self.dispatch_request()

  File "c:\\program files\\bazarr\\bazarr\\../libs\\flask\\app.py", line 1935, in dispatch_request

    return self.view_functions[rule.endpoint](**req.view_args)

  File "c:\\program files\\bazarr\\bazarr\\../libs\\flask_restful\\__init__.py", line 458, in wrapper

    resp = resource(*args, **kwargs)

  File "c:\\program files\\bazarr\\bazarr\\../libs\\flask\\views.py", line 89, in view

    return self.dispatch_request(*args, **kwargs)

  File "c:\\program files\\bazarr\\bazarr\\../libs\\flask_restful\\__init__.py", line 573, in dispatch_request

    resp = meth(*args, **kwargs)

  File "c:\\program files\\bazarr\\bazarr\\api.py", line 77, in wrapper

    return actual_method(*args, **kwargs)

  File "c:\\program files\\bazarr\\bazarr\\api.py", line 2018, in get

    result[\'episode\'] = int(guessit_result[\'episode\'])

TypeError: int() argument must be a string, a bytes-like object or a number, not \'list\
```
</details>

**Software**
- Bazarr Version: 0.9.10-beta.6
- Sonarr Version: 3.0.6.1266
- Radarr Version: 3.2.2.5080
- Operating System: Windows-10-10.0.19041-SP0
- Python Version: 3.8.0

**Additional Information**

This seems to be related to the subtitle filename. If I have `X.S01E01-E02.mkv` and `X.S01E01-E02.srt`, Bazarr won't process, but it will if I rename the subtitle to `X.S01E01.srt` and let Bazarr assign and rename the file as a multi later.

**Comment by fabiorzfreitas** [comment]
Yes, upload, sorry for not making it clear

**Comment by morpheus65535** [comment]
Should be fixed in upcoming 0.9.110-beta.7. Can you confirm please?

**Comment by fabiorzfreitas** [comment]
Seems to be working now, thank you!
